configs/common/packages.yaml

```diff
@@ -226,3 +226,9 @@
       variants: +python +grib2
     metplus:
       version: [4.1.0]
+    mapl:
```

Comment:
These packages have been defined earlier in the same file

Reply:
Are they? I thought they were too, but a find through this file reveals nothing. gftl-shared is there, but not MAPL or Yafyaml.

 https://github.com/NOAA-EMC/spack-stack/blob/develop/configs/common/packages.yaml

Reply:
Same confusion on my end! Good to have each of them now. We should probably list all of them in alphabetical order (in a separate PR that does nothing else).